ca-certificates/example-ca-cert.crt

```diff
@@ -0,0 +1,30 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This is an example ca certificate generated with https://github.com/FiloSottile/mkcert, adding just for git to preserve the `ca-certificates` folder, its just a public key, no harm or anything in having it in the repository